lib/ansible/config/base.yml

```diff
@@ -1715,7 +1715,7 @@ PARAMIKO_LOOK_FOR_KEYS:
   type: boolean
 PERSISTENT_CONTROL_PATH_DIR:
   name: Persistence socket path
-  default: ~/.ansible/pc
+  default: ~/.ansible/cp
```

Comment:
Not a typo, this was done to differentiate from the normal control path, 'persistence' here is about 'session' and not just about authentication (as is with the normal control path with ssh).

Reply:
I see, I'd never seen the "persistence control path", just the normal control path. My bad. Apologies for the noise :-)